reducer/src/main/java/com/graphicsfuzz/reducer/reductionopportunities/LiteralToUniformReductionOpportunity.java

```diff
@@ -25,18 +30,40 @@
 
   private final Expr literalExpr;
   private final PipelineInfo pipelineInfo;
+  private final TranslationUnit translationUnit;
+  private final int index;
 
-  LiteralToUniformReductionOpportunity(Expr literalExpr,
+  LiteralToUniformReductionOpportunity(TranslationUnit translationUnit,
+                                       Expr literalExpr,
+                                       int index,
                                        PipelineInfo pipelineInfo,
                                        VisitationDepth depth) {
     super(depth);
+
+    this.index = index;
     this.literalExpr = literalExpr;
     this.pipelineInfo = pipelineInfo;
+    this.translationUnit = translationUnit;
+  }
+
+  public Expr getLiteral() {
+    return this.literalExpr;
+  }
+
+  public int getIndex() {
+    return this.index;
   }
```

Comment:
I suggest removing these.  They shouldn't be needed.  If they are only used in tests then instead the test code should apply transformations and see that they have the right effect.

Reply:
Done.